src/world_inspector/mod.rs

```diff
@@ -388,13 +428,11 @@ impl<'a> WorldUIContext<'a> {
     }
 }
 
-/// Safety:
-/// The `location` must point to a valid archetype and index,
-/// and the function must have unique access to the components.
-pub(crate) unsafe fn try_display(
+pub(crate) fn try_display(
```

Comment:
This should still be `unsafe` because the `try_display` function requires the parameter to be valid in order to uphold memory safety.